tests/features/bootstrap/FeatureContext.php

```diff
@@ -598,8 +598,9 @@ public function iRestoreTheEnvironmentOfFromBackup($env, $site)
      */
     public function iRun($command)
     {
+        $terminus_bin = getenv('TERMINUS_BIN') ?: 'bin/terminus';
         $regex        = '/(?<!\.)terminus/';
-        $command = preg_replace($regex, sprintf('bin/terminus', $this->cliroot), $command);
+        $command = preg_replace($regex, getcwd() . '/' . $terminus_bin, $command);
```

Comment:
Perhaps I should have used `$this->cliroot` here, like the old code. (Note that `sprintf('bin/terminus', $this->cliroot)` is just `bin/terminus`, though; perhaps this was a typo.)